### ISSUE TITLE
ES3 support

### ISSUE THREAD

**Issue by medikoo** [WebReflection/event-target#6]
If I see [correctly](https://github.com/WebReflection/event-target/blob/master/index.js#L9) support for ES3 engines is not ensured.

(so `Every. JavaScript. Engine.` statement is not true ;-))

While it's not common to write for ES3 engines these days. It still can be the case in some scenarios, and `EventTarget` is core interface to many functionalities.

**Comment by WebReflection** [comment]
I don't just write things, I code cover 100% and test live on real targets:
http://webreflection.github.io/event-target/test/

**Comment by medikoo** [comment]
Ok, but what about ES3 compatible CJS module, that I `require` after I install package with `npm`.

There's none isn't it?

I write with CJS also for ES3 enviroments (one way for all)

**Comment by WebReflection** [comment]
> what about ES3 compatible CJS module, that I require after I install package with npm.

I have zero interest in complicating anyhow the maintenance of this project.

If you have a PR I might consider it, otherwise feel free to look elsewhere, I don't have time for non-bugs here, sorry.

**Comment by medikoo** [comment]
> If you have a PR I might consider it

Are you open for taking a PR, that makes it ES3 compatible out of a box? I may prepare one in such case

**Comment by medikoo** [comment]
> If you have a PR I might consider it, otherwise feel free to look elsewhere, I don't have time for non-bugs here, 

I wanted to comment at #4, but for some reason you closed ability for further communication there.

I didn't expect you to put more effort into that. I maintain myself hundreds of modules and also don't have time for most, so you don't have to explain this to me ;-)

I'm happy to code improvements myself, but before I invest my time, I need to know they're welcome, otherwise it'll be waste of time on my side.

**Comment by WebReflection** [comment]
you're complaining about things without even double checking:

  * you opened amp issue that wasn't existing
  * you are stating this module has side effects while ESM and CJS are poonyfills already without side effects
  * you are complaining about a de-facto standard such `.default` is these days
  * you insinuate I lied saying the code works as ES3 without checking the live test page

When you'll be able to ask things in a friendly way without insinuations, moaning about ESM vs CJS, and actually testing before opening bugs, then I'll welcome you and neither you, nor me, would ever waste any time at all.

**Comment by medikoo** [comment]
>you are stating this module has side effects while ESM and CJS are poonyfills already without side effects

I didn't state it anywhere. I responded that I don't want side effect, when you recommend to me to load it in a way that implies side effects

> you are complaining about a de-facto standard such .default is these days

It's just your opinion. Mind that many people rely on purely on CJS, while ESM is still not usable in most environments, and it looks it will never be as usable as CJS is today. I'm off on ESM for that very reason.

> you insinuate I lied saying the code works as ES3 without checking the live test page

As I indicated, I was after CJS module which works (with no side effecs) when bundled into ES3 enviromnment, simple as that.

Communication will be way easier, if you would try to hear what other party wants to say :)

Now feel free to close this issue for further comments as well ;-)
